sklearn/linear_model/tests/test_logistic.py

```diff
@@ -89,6 +91,30 @@ def test_error():
         assert_raise_message(ValueError, msg, LR(max_iter="test").fit, X, Y1)
 
 
+def test_logistic_cv_neg_mean_squared_error():
```

Comment:
It's a bit awkward that a test by this name does not check that the provided score is maximised. I assume there is an existing test that does. Either merge this test into that one, or make it explicit that this test is about the score method.